CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Minor: Remove TwitchEmotes.com attribution and the open/copy options when right-clicking a Twitch Emote. (#2214, #3136)
 - Minor: Strip leading @ and trailing , from username in /user and /usercard commands. (#3143)
 - Minor: Display a system message when reloading subscription emotes to match BTTV/FFZ behavior (#3135)
+- Minor: Subcategories in settings are now searchable. (#2707, #3157)
```

Comment:
As agreed with pajlada, we're now against putting issue numbers into changelog entries. CC @felanbird

```suggestion
- Minor: Subcategories in settings are now searchable. (#3157)
```